app/src/main/java/com/example/hris/ui/viewmodels/AddTimeLogsViewModel.kt

```diff
@@ -0,0 +1,38 @@
+package com.example.hris.ui.viewmodels
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.viewModelScope
+import com.example.hris.model.ResponseModel
+import com.example.hris.repository.HrisRepository
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+@HiltViewModel
+class AddTimeLogsViewModel @Inject constructor(
+    private val hrisRepository: HrisRepository,
+    application: Application
+) : AndroidViewModel(application) {
+
+    val loadingDialogState = MutableLiveData<Boolean>()
+    val callValue = MutableLiveData<ResponseModel>()
```

Comment:
ProTip: You don't need the contents of the `ResponseModel` for this one. To save memory, you can just make this `liveDataSuccess = MutableLiveData<Unit>()` then when the API call succeeds, you can just do `liveDataSuccess.value = Unit`